<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>whoisForm</class>
 <widget class="QWidget" name="whoisForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>550</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>167772</width>
    <height>167772</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Whois application</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>URW Palladio L</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>
  Whois domain name lookup 
</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Search for                 </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="input">
          <property name="text">
           <string comment="Enter IP address or hostname"/>
          </property>
          <property name="maxLength">
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Whois server</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="serverBox">
          <property name="editable">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>whois.afrinic.net</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>whois.apnic.net</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>whois.arin.net</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>whois.lacnic.net</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>whois.ripe.net</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="whoisButton">
          <property name="text">
           <string>Whois</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QGridLayout" name="gridLayout_3"/>
        </item>
        <item>
         <widget class="QRadioButton" name="ipv4Button">
          <property name="text">
           <string>IPv4</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="ipv6Button">
          <property name="text">
           <string>IPv6</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>348</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QTextEdit" name="output">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="clearButton">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="quitButton">
          <property name="text">
           <string>Quit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
